ResearchKit/Consent/ORKVisualConsentStepViewController.m

```diff
@@ -506,10 +512,24 @@ - (void)observedScrollViewDidScroll:(UIScrollView *)scrollView {
 }
 
 - (void)showViewController:(ORKConsentSceneViewController *)viewController forward:(BOOL)forward animated:(BOOL)animated {
+    [self showViewController:viewController forward:forward animated:animated preloadNextViewController:YES];
+}
+
+- (void)showViewController:(ORKConsentSceneViewController *)viewController forward:(BOOL)forward animated:(BOOL)animated preloadNextViewController:(BOOL)preloadNextViewController {
     [self showViewController:viewController
                      forward:forward
                     animated:animated
-                  completion:nil];
+                  completion:^(BOOL finished) {
+                      if (preloadNextViewController) {
+                          ORKConsentSceneViewController *nextViewController = [self viewControllerForIndex:[self currentIndex]+1];
```

Comment:
This code is pretty ugly. We should not have to create an entire view controller in order to add an image to the tint cache, especially an image that just comes from a model object that we already have at this level.